### PR TITLE
Update tshoot_access-browser.html

### PR DESCRIPTION
**Pull request by ygyryn**
## This PR is a:

- [ ] New topic
- [x] Content update
- [x] Content fix or rewrite
- [x] Bug fix or improvement

## Summary

When this pull request is merged, it will...
be created new "tshoot_access-browser.html" template for 2.3 version, in which will be fixed outdated text for magento 2.3 support PHP version in the section:
"Symptom: The following message displays when you try to access the Magento storefront or Admin"

Old outdated text:
Whoops, it looks like you have an invalid PHP version.
Magento supports PHP 5.5 or newer.

New text, that is actual for current version:
Magento supports PHP 7.1.3 or later. Please read Magento System Requirements.

## Additional information

List all affected URLs 
https://devdocs.magento.com/guides/v2.3/install-gde/trouble/tshoot_access-browser.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/install-gde/trouble/tshoot_access-browser.md [hunk 1]
```diff
@@ -1 +1,40 @@
-../../../v2.2/install-gde/trouble/tshoot_access-browser.md
```
Comment (jcalcaben) [inline guides/v2.3/install-gde/trouble/tshoot_access-browser.md]: This is a symlink file. Replacing the contents of this file will break the build process.

To properly update this version of the file, please delete the symlink file and create a new file with the updated content.

Comment (ygyryn) [reply guides/v2.3/install-gde/trouble/tshoot_access-browser.md]: @jcalcaben 
Thank you for explaining.
Done.

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @ygyryn!

**Pull request merged**